pytket/extensions/qiskit/backends/aer.py

```diff
@@ -80,8 +80,8 @@
 )
 
 if TYPE_CHECKING:
-    from qiskit.providers.aer import AerJob  # type: ignore
-    from qiskit.providers.aer.backends.aerbackend import AerBackend as QiskitAerBackend  # type: ignore
+    from qiskit_aer import AerJob
```

Comment:
Does `AerJob` still need to be inside the `if TYPE_CHECKING:`?

Reply:
From my understanding we are not using it besides type annotations?

Reply:
Ah okay, that seems fine to me then :)